examples/deployments/ECS/README.md

```diff
@@ -0,0 +1,97 @@
+|-|-|-|-|
+|2379|TCP|2379|HTTP|
+|2380|TCP|2380|HTTP|
+- Environment variable key is `ALLOW_NONE_AUTHENTICATION` and value is `YES`
+2. NATS container
+- Container name use `nats`
+- Image URL is `nats` and **Yes** for Essential container
+- Container port  
+
+|Container port|Protocol|Port name| App protocol|
+|-|-|-|-|
+|4222|TCP|4222|HTTP|
+|6222|TCP|6222|HTTP|
+|8222|TCP|8222|HTTP|
+- Docker configuration, add `-js, --trace` in **Command**
+
+## 3. vLLM Task Definitions Setup
+1. Dynamo vLLM Frontend Task
+This task will create vLLM frontend, processors, routers and a decode worker.
+Please follow steps below to create this task
+- Set container name as `dynamo-frontend` and use prebuild [Dynamo container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/ai-dynamo/containers/vllm-runtime).
+- Choose `Amazon EC2 instances` as the **Launch type** with **Task size** `2 vCPU` and `40 GB`memory
+- Choose `host` as the Network mode. 
+- Container name use `dynamo-vLLM-frontend`
+- Add your Image URL (You can use the prebuild [Dynamo container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/ai-dynamo/containers/vllm-runtime)) and **Yes** for Essential container. It can be AWS ECR URL or Nvidia NGC URL. If using NGC URL, please also choose **Private registry authentication** and add your Secret Manager ARN or name. 
+- Container port  
+
+|Container port|Protocol|Port name| App protocol|
+|-|-|-|-|
+|8000|TCP|8000|HTTP|
+- Use `1` GPU for **Resource allocation limits**
+- Environment variables settings as below. Will override the `IP_ADDRESS` later.
+
+|Key|Value type|Value|
+|-|-|-|
+|ETCD_ENDPOINTS|Value|http://IP_ADDRESS:2379|
+|NATS_SERVER|Value|nats://IP_ADDRESS:4222|
+- Docker configuration  
+Add `sh,-c` in **Entry point** and `cd components/backends/vllm && python -m dynamo.frontend --router-mode kv & python3 -m dynamo.vllm --model Qwen/Qwen3-0.6B --enforce-eager` in **Command**
+
+2. Dynamo vLLM PrefillWorker Task
+Create the PrefillWorker task same as the frontend worker, except for following changes
+- Set container name as `dynamo-prefill`
+- No container port mapping
+- Docker configuration with command `cd components/backends/vllm && python3 -m dynamo.vllm --model Qwen/Qwen3-0.6B --enforce-eager --is-prefill-worker`
+
+## 4. Task Deployment
+You can create a service or directly run the task from the task definition
+1. ETCD/NATS Task
+- Choose the Fargate cluster for **Existing cluster** created in the hello world example.
+- Wait for this deployment to finish, and get the **Private IP** of this task. 
+2. Dynamo Frontend Task
+- Choose the EC2 cluster for **Existing cluster** created in step 1.
+- In the **Container Overrides**, use the IP for ETCD/NATS task for the `ETCD_ENDPOINTS` and `NATS_SERVER` values.
+3. Dynamo PrefillWorker Task
+- Choose the EC2 cluster for **Existing cluster** created in step 1.
+- In the **Container Overrides**, use the IP for ETCD/NATS task for the `ETCD_ENDPOINTS` and `NATS_SERVER` values.
```

Comment:
What is "Container Overrides" referring to? I don't see that as a field in the json files